js/repl/loadPlugin.js

```diff
@@ -19,16 +20,19 @@ export default function loadPlugin(
   const base = config.baseUrl || "https://bundle.run";
   const url = `${base}/${config.package}@${config.version || ""}`;
 
-  loadScript(url, success => {
-    if (success) {
-      state.isLoaded = true;
-      state.isLoading = false;
-      state.plugin = window[camelCase(state.config.package)];
```

Comment:
I thought we might use this but we never did so I removed it in this PR for simplicity.